src/vi/chatparser/chatparser.py

```diff
@@ -131,8 +131,6 @@ def _lineToMessage(self, line, roomname):
             message.status = states.IGNORE
             return message
 
-        for char in ("*", "?", ",", "!"):
-            text = text.replace(char, "")
```

Comment:
Not sure why this code was here; looks like leftovers from a prior refactoring.  The 'text' variable is never used after this substitution.